index.html

```diff
@@ -4,9 +4,15 @@
 	<meta charset="UTF-8">
 	<meta http-equiv="X-UA-Compatible" content="IE=edge">
 	<meta name="viewport" content="width=device-width, initial-scale=1.0">
-	<title>Document</title>
+	<title>We rock!</title>
 </head>
 <body>
+
+	<h1>Lorem ipsum dolor sit amet consectetur adipisicing elit. Mollitia, voluptas laborum consequuntur sunt temporibus ab adipisci consectetur molestias odio quae voluptate ea dolore nostrum omnis reiciendis aliquam asperiores blanditiis! Laudantium.</h1>
```

Comment:
Really love the usage of a semantic heading - perhaps consider putting this within a `header` element.

Reply:
great tip, this is fixed.